features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumRawEvent.kt

```diff
@@ -212,16 +211,9 @@ internal sealed class RumRawEvent {
 
     internal data class WebViewEvent(override val eventTime: Time = Time()) : RumRawEvent()
 
-    internal data class SendTelemetry(
-        val type: TelemetryType,
-        val message: String,
-        val stack: String?,
-        val kind: String?,
-        val coreConfiguration: TelemetryCoreConfiguration?,
-        val additionalProperties: Map<String, Any?>?,
-        val onlyOnce: Boolean = false,
-        override val eventTime: Time = Time(),
-        val isMetric: Boolean = false
+    internal data class TelemetryEventWrapper(
```

Comment:
we probably can keep the same name for the class - `SendTelemetry`. but not a big deal